src/edu/csus/ecs/pc2/ui/MultipleDataSetPane.java

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
```

Comment:
This PR is about adding support for submitting judge's solutions from a CDP.  The changes in this file have to do with adding UI support for loading sample test data cases, which has little or nothing to do with submitting judge's solutions and is completely outside the scope of this PR.  As someone who has been (appropriately) chastised for adding even a few lines of code in a PR which are not related to the specification laid out in the PR and its corresponding Issue, I object to the inclusion of 60+ lines of new code that are not related to submitting judge's solutions.  Please revert/remove these changes and submit them as part of a PR for the appropriate issue.

Reply:
I consider this an approval showstopper.